scenarios/kubernetes_e2e.py

```diff
@@ -313,28 +339,15 @@ def main(args):
     cluster = args.cluster or 'e2e-gce-%s-%s' % (
         os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0))
 
-    if args.kubeadm:
+    if args.kubeadm is not None:
```

Comment:
`if not args.kubeadm`

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations